src/api/sso/src/slack-bolt-client.js

```diff
@@ -0,0 +1,77 @@
+const { logger } = require('@senecacdot/satellite');
+const { App } = require('@slack/bolt');
+
+const { CHANNEL_ID } = process.env;
+const createSlackApp = () => {
+  const { SLACK_BOT_TOKEN, SIGNING_SECRET } = process.env;
+  if (!(SLACK_BOT_TOKEN && SIGNING_SECRET && CHANNEL_ID)) {
+    return false;
```

Comment:
Maybe add a `logger.info('Skipping Slack Bot integration, not configured')` so it's clear that this isn't running.